.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+# This workflows will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+name: Publish Python Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USER }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASS }}
+      run: |
+        python setup.py sdist bdist_wheel
```

Comment:
We don't need the bdist_wheel do we?

Reply:
I think so, but I'm not sure. The metpy example below relies on the output in the ./dist folder. So I think that's a result of the building a wheel. 

Are there disadvantages to building wheels like that?

Reply:
I am unsure :( I have always just done `sdist` since I have no compiling necessary within my code, which I thought wheels were for.

Reply:
I know bdist_wheel works with "pure python" packages, so I suggest we stick with this for now